js/util-galery.js

```diff
@@ -0,0 +1,68 @@
+
+const bigPhoto = document.querySelector('.big-picture');
+const body = document.querySelector('.body');
+const commentCounter = document.querySelector('.social__comment-count');
+const commentsLoader = document.querySelector('.comments-loader');
+const closeButton = document.querySelector('.big-picture__cancel  cancel');
+const socialComments = document.querySelector('.social__comments');
+
+const hideBigPhoto = () => {
+  bigPhoto.classList.add('hidden');
+  body.classList.remove('modal-open');
+  commentCounter.classList.remove('hidden');
+  commentsLoader.classList.remove('hidden');
+  //document.removeEventListener('keydown', onDocumentKeydown);
+  //closeButton.removeEventListener('click', onCloseButton);
+};
+
+const onDocumentKeydown = (evt) => {
```

Comment:
Нужно заменить на обычную функцию, чтобы она "всплывала" наверх и была доступна для удаления выше в нашей функции `hideBigPhoto`, это самый простой способ сделать функцию доступной выше места где она была определена
```suggestion
function onDocumentKeydown (evt) {
  if (evt.key === 'Escape') {
    evt.preventDefault();
    hideBigPhoto();
  }
}
```